src/shims/shims.proj

```diff
@@ -43,31 +42,23 @@
 
     <Exec Command="$(GenFacadesCmd) -contracts:&quot;@(NETStandardContracts)&quot; @&quot;$(GenFacadesResponseFile)&quot;" WorkingDirectory="$(ToolRuntimePath)" />
 
-    <!-- Copy the facades to the package ref and lib folders to be included in the packages -->
-    <!-- TODO: replace with BinPlacing targets -->
-    <ItemGroup>
-      <PackageOutputPaths Include="$(BinDir)pkg/$(TargetGroup)/ref" Condition="'$(_bc_TargetGroup)'!='uapaot'"/>
-      <PackageOutputPaths Include="$(BinDir)pkg/uap/ref" Condition="'$(_bc_TargetGroup)'=='uapaot'"/>
-      <PackageOutputPaths Include="$(BinDir)pkg/$(TargetGroup)/lib" />
-      <PackageOutputPaths Include="$(RefPath)" />
-      <PackageOutputPaths Include="$(RuntimePath)" />
```

Comment:
See https://github.com/dotnet/corefx/blob/master/binplace.targets#L31 they will till be put into the RuntimePath.